src/spikeinterface/comparison/comparisontools.py

```diff
@@ -132,62 +132,6 @@ def compute_matching_matrix(
         num_units_train2,
         delta_frames,
     ):
-        """
```

Comment:
Why did you remove the docstring from here?

Adding the other is great but what is the gain of having less?

Reply:
done